src/controllers/userController.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+let idTracker = 1;
+
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const isUser = (userId) => {
+  return users.some(user => user.id === userId);
+};
+
+const validateId = (req, res, next) => {
+  const { id } = req.params;
+
+  if (!id || isNaN(+id)) {
```

Comment:
You are converting the 'id' to a number using '+id'. Although it's a valid way of doing it, it can lead to confusion and errors. Consider using 'parseInt()' for clarity and better error handling.